memory_profiler.py

```diff
@@ -111,12 +143,7 @@ def ps_util_tool():
                 else 'get_memory_info'
             mem = getattr(process, meminfo_attr)()[0] / _TWO_20
             if include_children:
-                try:
-                    for p in process.children(recursive=True):
-                        mem += getattr(p, meminfo_attr)()[0] / _TWO_20
-                except psutil.NoSuchProcess:
-                    # https://github.com/fabianp/memory_profiler/issues/71
-                    pass
+                mem +=  sum(_get_child_memory(process, meminfo_attr))
```

Comment:
Apologies for the direct approach, I wanted to point out a possible misreport on the the total memory usage when multiprocessing. 

My thought was that I could save some time from everyone by using this open issue. I can go into more details or a new issue, if this does not suffice.